2019-04-11 18:44:32,758 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:32,861 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:32,952 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:33,059 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:33,123 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:33,186 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:33,281 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:33,381 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:58,176 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-11 18:44:58,184 - [updatemp.py:108] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:/GitHub/wechat_sogou_crawl/updatemp.py", line 60, in <module>
    temp_qunfa_id = int(wz_item['qunfa_id'])
TypeError: string indices must be integers
2019-04-23 17:41:20,307 - [api.py:93] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\api.py", line 88, in search_gzh_info
    text = self._get(url_temp, referer=request_url)  # 需要优化
  File "D:\GitHub\wechat_sogou_crawl\wechatsogou\basic.py", line 136, in _get
    raise WechatSogouVcodeException('weixin.sogou.com verification code')
wechatsogou.exceptions.WechatSogouVcodeException: weixin.sogou.com verification code
2019-04-23 17:41:20,416 - [updatemp.py:108] - ERROR - Exception Logged
Traceback (most recent call last):
  File "D:/GitHub/wechat_sogou_crawl/updatemp.py", line 60, in <module>
    temp_qunfa_id = int(wz_item['qunfa_id'])
TypeError: string indices must be integers
